modelforge/potential/physnet.py

```diff
@@ -665,8 +665,9 @@ class PhysNet(BaseNetwork):
         Number of interaction residual blocks.
     number_of_modules : int
         Number of PhysNet modules.
-    activation_function : str
-        Name of the activation function to use.
+    activation_function_parameter : Dict
+        Dict that contains keys: activation_function_name [str], activation_function_arguments [Dict],
+        and activation_function [Type[torch.nn.Module]].
```

Comment:
Oh, I was under the impression that the pydantic model takes the activation_function_paramter dict and returns an instance of the activation function?

Reply:
It will take in the name and any possible parameters, and then set an instance of the class (with appropriate parameters) to a variable "activation_function"; since the pydantic model is hierarchical, the "activation_function" variable ends up a level down ( `core_parameter.activation_function_parameter.activation_function`).  Right now, the way the code is structured, we are dumping the pydantic model to a dict and then passing that to the NNP class using  argument unpacking (rather than parsing the model and passing individual arguments), rather than passing the pydantic model itself.  So having this accept the dict seemed like the fastest/least disruptive to the rest of the code, way to handle this.  

I think ultimately we absolutely should be passing the pydantic models instead, but I think we should wait until after the first release is cut.